src/puppetlabs/ezbake/core.clj

```diff
@@ -406,7 +426,8 @@ Dependency tree:
                                                           :main-namespace
                                                           "puppetlabs.trapperkeeper.main")
          :java-args                 (get-local-ezbake-var lein-project :java-args
-                                                          "-Xmx192m")}))))
+                                                          "-Xmx192m")
+         :bootstrap-source          (get-local-ezbake-var lein-project :bootstrap-source :file)}))))
```

Comment:
Reading through the code, it kind of seems like if a user put in any value besides 'directory' for this setting, it'd just silently ignore it and default to `file`.  Seems like maybe we should wrap a little schema validation around this line and add a test that covers that.

Also, I don't know what others think, but I think I'd kind of prefer `bootstrap-cfg` and `services-d` for the legal values here.  They're not entirely intuitive, but I don't know that these values are either.  `directory` seems kind of misleading since if you pick it, you're actually getting set up with 2 directories.  I don't have a super strong opinion about this, though, so I'm ok with it like this, but would like to see if @nathanielksmith or anyone else has thoughts.